Source/Scene/Scene.js

```diff
@@ -2468,7 +2468,7 @@ define([
             scene._fxaa.clear(context, passState, clearColor);
         }
 
-        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR) {
+        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR && !picking) {
```

Comment:
I'm not sure this is the right fix. The condition should always evaluate to false when picking. All of the environment should not render and all post processing should be disabled when `frameState.pick` is `true`.